app/services/bookings/gitis/fake_crm.rb

```diff
@@ -46,10 +46,14 @@ def create_entity(entity_id, data)
         end
       end
 
-      data.keys.each do |key|
+      data.each do |key, value|
         unless ALLOWED.include?(key)
           raise "Bad Response - attribute '#{key}' is not recognised"
         end
+
+        if key =~ /@odata.bind\z/ && (value.blank? || value !~ Bookings::Gitis::Entity::BIND_FORMAT)
```

Comment:
Minor: is the `value.blank?` necessary here? If it is blank it'll match the inverted regex and give the same result?